lib/ood_core/job/adapters/lsf/batch.rb

```diff
@@ -141,6 +141,7 @@ def cluster_args
     # Call a forked Lsf command for a given cluster
     def call(cmd, *args, env: {}, stdin: "")
       cmd = OodCore::Job::Adapters::Helper.bin_path(cmd, bindir, bin_overrides)
+      cmd = OodCore::Job::Adapters::Helper.ssh_wrap(cmd, submit_host)
```

Comment:
When calling `o, e, s = Open3.capture3(env, cmd, *args, stdin_data: stdin.to_s)`, when executing via ssh it would be preferable if cmd is be `ssh` and args has the rest of the arguments, one per array element. This may avoid some potential problems with placing quotes in the correct place for arguments.

Instead of wrapping cmd to build a larger string, you would have code that does something similar to what the Linux host adapter does. Though that adapter builds an array of arguments, where the first element is the command, and then it passes it to call. Here, you will want to potentially produce new cmd and args (instead of just cmd). One approach is to use multiple assignment for that:

```
irb(main):001:0> a,b = [1,2]
=> [1, 2]
irb(main):002:0> a
=> 1
irb(main):003:0> b
=> 2
```